### DISCUSSION TITLE
Oracle Database

### DISCUSSION THREAD

**Discussion by gasparini2016** [frangoteam/FUXA#1499] [Q&A]
Does anyone have any idea how I could record data collected by Fuxa into an Oracle database? ODBC? API?

**Comment by MatthewReed303** [comment]
@gasparini2016 you would have to install Oracle ODBC driver, add the installation to this script https://github.com/frangoteam/FUXA/blob/master/odbc/install_odbc_drivers.sh and add to https://github.com/frangoteam/FUXA/blob/master/odbc/odbcinst.ini
